test/stdlib/UnicodeScalarPropertiesTests.swift

```diff
@@ -248,4 +252,36 @@ UnicodeScalarPropertiesTests.test("properties.numericTypeAndValue") {
   expectEqual(4, us("\u{2463}").properties.numericValue)
 }
 
+// We can't reliably test the `supportedVersion` property because its value
+// changes depending on the version of ICU linked. We can at least sanity-check
+// the `Version` structure, though.
```

Comment:
Well, we _could_ test that it doesn't return something obviously wrong (like a major version <= 0).